lib/rouge/lexers/systemd.rb

```diff
@@ -27,7 +27,7 @@ class SystemD < RegexLexer
         rule %r/(.*?)(\\\n)/ do
           groups Text, Str::Escape
         end
-        rule %r/(.*)'/, Text, :pop!
+        rule %r/(.*)'?/, Text, :pop!
```

Comment:
I opted to add a line continuation to the existing sample. It looked like each lexer had a single example file and since Systemd doesn't really accept more than one proper service definition, I thought this was the clearest choice.